textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/DoubleField.java

```diff
@@ -0,0 +1,47 @@
+package edu.uci.ics.textdb.common.field;
+
+import edu.uci.ics.textdb.api.common.IField;
+
+public class DoubleField implements IField{
+    
+    private Double value;

```

Comment:
I think it will be fine here since we want to support all data types. Another easier way is to store `value` as primitive types. (e.g. `int` `double`...), only package them in the `getValue()` function. It could save objects creation a bit especially when `getValue()` is not called.